go-app/main.go

```diff
@@ -172,3 +176,63 @@ func loadAnalyseLog(w http.ResponseWriter, r *http.Request) {
 	}
 	reportTempl.Execute(w, report.FullLogDetails.Analysis_details)
 }
+func loadEventDetails(w http.ResponseWriter, r *http.Request) {
+	r.ParseMultipartForm(10 << 20)
```

Comment:
What are the 10 << 20 constants used for?

Reply:
That means that the size of the log file won't be bigger than 10 MB